packages/mdc-list/README.md

```diff
@@ -80,6 +80,12 @@ attached to the root element, or the element has an ancestor with class `mdc-the
 </html>
 ```
 
+#### Non-interactive lists
+
+By default, list items receive styles for hover, focus, and press states (including the ripple effect if
+`MDCRipple` is instantiated). It is possible to opt out of these styles by adding the `mdc-list--non-interactive`
```

Comment:
Should we mention that the focus styles only apply if the element is one that
naturally receives focus (`<a href="">`, `<button>`, etc.) or has `tabindex="0"`?
Or do we think most clients figure that out?

Reply:
I'll add a comment.